content/docs/reference/service-urls.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Service URLs
+id: service-urls
+description: This page covers all of Pomerium's service URL settings, including authenticate and authorize external and internal service URLs.
+keywords:
+  - authenticate service url
+  - authenticate internal service url
+  - authenticate_service_url
+  - authenticate_internal_service_url
+  - authorize service url
+  - authorize_service_url
+  - authorize internal service url
+  - authorize_internal_service_url
+pagination_prev: null
+pagination_next: null
+toc_max_heading_level: 2
+---
+
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem'; import PomeriumAuthenticateServices from '@site/content/docs/admonitions/\_pomerium-authenticate-services.mdx';
+
+# Service URL Settings
+
+## Authenticate Service URL
+
+The **Authenticate Service URL** setting defines the externally accessible URL where Pomerium redirects end users (clients) to authenticate against an identity provider.
```

Comment:
I think we should note here that this is only required if not using hosted authenticate.

Reply:
I think it could be helpful to add a little introduction (before any of the section headings) to spell out what these settings are for:
 - The authenticate service URL controls whether Pomerium will use the hosted authenticate service or not. If using your own IdP you will need this URL when configuring your IdP client (for the OAuth callback URL).
 - The other service URL settings are only for split service mode.

Reply:
I've added a few suggestions for tweaks here. I know I mentioned talking about the OAuth callback URL above but I'm thinking maybe it's better off here.